doc/en/contents.rst

```diff
@@ -10,32 +10,51 @@ Full pytest documentation
 .. toctree::
    :maxdepth: 2
 
-   overview
-   apiref
-   example/index
+   getting-started
```

Comment:
i recall @pfctdayelise working on the structure, perhaps we can use the pictures and merge the docs branch 

also how far are those appart